modules/using-a-drift-metric-in-a-credit-card-scenario.adoc

```diff
@@ -0,0 +1,204 @@
+= Using a drift metric in a credit card scenario
+
+[role='_abstract']
+This example scenario deploys an XGBoost model into your cluster and reviews its output using a drift metric.
+
+ The XGBoost model was created for the purpose of this demonstration and predicts credit card approval based on the following features: age, credit score, years of education, and years in employment. 
+
+Once the model is deployed and the data that we upload is formatted, we'll use the mean shift metric to monitor for data drift. This metric is useful for ensuring that a model remains accurate and reliable in a production environment.
+
+Mean shift compares a numeric test dataset against a numeric training dataset, and produces a p-value measuring the probability that the test data came from the same numeric distribution as the training data. A p-value less than 0.05 indicates a statistically significant drift between the two. A p-value equal to or greater than 0.05 indicates no statistically significant evidence of drift.
+
+[NOTE]
+--
+Mean shift performs best when each feature in the data is normally distributed. Choose a different metric for working with different or unknown data distributions.
+--
+
+.Prerequisites
+
+ifdef::upstream[]
+* Your cluster administrator added you as a user to the cluster and link:{odhdocshome}/monitoring-data-science-models/#configuring-trustyai_monitor[configured TrustyAI] for use in the project that contains the deployed models.
+
+* You set up TrustyAI for your project, as described in link:{odhdocshome}/monitoring-data-science-models/#setting-up-trustyai-for-your-project_monitor[Setting up TrustyAI for your project].
+endif::[]
+//downstream
+ifndef::upstream[]
+* Your cluster administrator added you as a user to the cluster and link:{rhoaidocshome}{default-format-url}/monitoring_data_science_models/configuring-trustyai_monitor[configured TrustyAI^] for use in the project that contains the deployed models.
+
+* You set up TrustyAI for your project, as described in link:{rhoaidocshome}{default-format-url}/monitoring_data_science_models/setting-up-trustyai-for-your-project_monitor[Setting up TrustyAI for your project^].
+endif::[]
+
+
+.Procedure
+
+. Obtain a bearer token to authenticate your external endpoints by running the following command:
++
+[source, bash]
+----
+oc apply -f resources/service_account.yaml
+export TOKEN=$(oc create token user-one)
+----
+
+. In your model namespace, deploy the storage container, serving runtime, and the credit model: 
++
+[source, bash]
+----
+oc project model-namespace || true
+oc apply -f resources/model_storage_container.yaml
+oc apply -f resources/odh-mlserver-1.x.yaml
+oc apply -f resources/model_gaussian_credit.yaml
+----
+
+. Set the route for your data upload:
++
+[source, bash]
+----
+TRUSTY_ROUTE=https://$(oc get route/trustyai-service --template={{.spec.host}})
+----
+
+. Download the training data payload (472 KB):
```

Comment:
Maybe helpful to mention what 472 KB refers to. Maybe it's obvious to others but new to me so wanted to call out!